docs/public-documentation/Trigger.md

```diff
@@ -9,6 +9,175 @@ For information on setup and configuration details, see the overview
 
 # Examples
 
+## C#
+
+The C# function can be created using one of the following C# modes:
+
+- [In-process class library](https://docs.microsoft.com/en-us/azure/azure-functions/functions-dotnet-class-library): compiled C# function that runs in the same process as the Functions runtime.
+- [Isolated process class library](https://docs.microsoft.com/en-us/azure/azure-functions/dotnet-isolated-process-guide): compiled C# function that runs in a process isolated from the runtime. Isolated process is required to support C# functions running on .NET 5.0.
+
+The following example shows a C# function that reads and logs the Kafka message as a KafkaEvent:
+
+### InProcess
+The following example shows a C# function that reads and logs the Kafka message as a KafkaEvent:
+
+```csharp
+[FunctionName("KafkaTrigger")]
+public static void Run(
+    [KafkaTrigger("BrokerList",
+                    "topic",
+                    Username = "ConfluentCloudUserName",
+                    Password = "ConfluentCloudPassword",
+                    Protocol = BrokerProtocol.SaslSsl,
+                    AuthenticationMode = BrokerAuthenticationMode.Plain,
+                    ConsumerGroup = "$Default")] KafkaEventData<string> kevent, ILogger log)
+{            
+    log.LogInformation($"C# Kafka trigger function processed a message: {kevent.Value}");
+}
+```
+
+
+### IsolatedProcess
+```csharp
+[Function("KafkaTrigger")]
+public static void Run(
+    [KafkaTrigger("BrokerList",
+                    "topic",
+                    Username = "ConfluentCloudUserName",
+                    Password = "ConfluentCloudPassword",
+                    Protocol = BrokerProtocol.SaslSsl,
+                    AuthenticationMode = BrokerAuthenticationMode.Plain,
+                    ConsumerGroup = "$Default")] string eventData, FunctionContext context)
+{
+    var logger = context.GetLogger("KafkaFunction");
+    logger.LogInformation($"C# Kafka trigger function processed a message: {JObject.Parse(eventData)["Value"]}");
+}
+```
+
+## Java
+
+The following example shows an Kafka trigger binding which logs the message body of the Kafka trigger.
+
+```java
+@FunctionName("KafkaTrigger")
+public void runSingle(
+        @KafkaTrigger(
+            name = "KafkaTrigger",
+            topic = "topic",  
+            brokerList="%BrokerList%",
+            consumerGroup="$Default", 
+            username = "%ConfluentCloudUsername%", 
+            password = "ConfluentCloudPassword",
+            authenticationMode = BrokerAuthenticationMode.PLAIN,
+            protocol = BrokerProtocol.SASLSSL,
+            dataType = "string") String kafkaEventData,
+        final ExecutionContext context) 
+    {
+        context.getLogger().info(kafkaEventData);
+    }
+```
+
+## Javascript
+
+The following example shows a Kafka trigger binding in a function.json file and a JavaScript function that uses the binding. The function reads and logs a Kafka  message.
```

Comment:
Please add the link with JavaScript function -- https://docs.microsoft.com/en-us/azure/azure-functions/functions-reference-node
This will be helpful if any new customer is directly coming on kafka trigger docs and missed exploring the main

Reply:
Just validated it's in rabbitmq doc also